mmseg/datasets/dataset_wrappers.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, datasets, separate_eval=True):
         assert separate_eval in [True, False], \
             f'separate_eval can only be True or False,' \
             f'but get {separate_eval}'
+        self.have_same_type = \
```

Comment:
As this attribution type is bool, so I prefer to `is_same_type` instead of `have_same_type`

Reply:
well, renamed